README.md

```diff
@@ -48,6 +48,8 @@ By default, the ipam-plugin assumes that the "Cloud Network Automation" licensed
 
 Plugin is installed by pulling the infoblox/ipam-plugin from the docker store and setting its environment variables.
 
+In Docker swarm mode the plugin needs to be installed on all the nodes.
```

Comment:
You might want to make it clear that the following are "options" (i.e. not two "steps")